libnavigation-core/src/main/java/com/mapbox/navigation/core/preview/RoutesPreviewController.kt

```diff
@@ -0,0 +1,144 @@
+package com.mapbox.navigation.core.preview
+
+import androidx.annotation.UiThread
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+import com.mapbox.navigation.base.route.NavigationRoute
+import com.mapbox.navigation.core.routealternatives.mapToMetadata
+import com.mapbox.navigator.RoutesData
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.sync.Mutex
+import kotlinx.coroutines.sync.withLock
+import java.util.concurrent.CopyOnWriteArrayList
+
+@ExperimentalPreviewMapboxNavigationAPI
```

Comment:
```suggestion
```

same here, internal does not need it.

but not a big deal could go either way

Reply:
done